paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc

```diff
@@ -215,14 +215,7 @@ class ConvMKLDNNHandlerT
        * ('any') which lets a primitive (convolution in this case) choose
        * the memory format preferred for best performance
        */
-      // TODO(jczaja): This is workaround to make grad op UT's numerical
-      // gradient computation proper as this op is called directly without
-      // fetch op following it , so numercial grad is computed (in python)
-      // using block formats which will give wrong results
-      const std::string data_format = ctx.Attr<std::string>("data_format");
-      auto chosen_memory_format =
-          is_test ? MKLDNNMemoryFormat::any
-                  : platform::data_format_to_memory_format(data_format);
+      auto chosen_memory_format = MKLDNNMemoryFormat::any;
 
       // Check the format for user's special output
       if (chosen_memory_format != MKLDNNMemoryFormat::any) {
```

Comment:
Is it needed, since we set `chosen_memory_format` earlier?

Reply:
good catch! I deleted dead section of code